packages/devtools_app/lib/src/screens/memory/memory_controller.dart

```diff
@@ -419,7 +419,7 @@ class MemoryController extends DisposableController
 
   /// Source of memory heap samples. False live data, True loaded from a
   /// memory_log file.
-  bool offline = false;
+  final offline = ValueNotifier<bool>(false);
```

Comment:
I'm not seeing anywhere where we are actually listening to this notifier. Are we anywhere? If that is the case, there is no added value gained by converting to a ValueNotifier instead of a bool.

Reply:
It is passed to MemoryTimeline, that needs up-to-date value:
https://github.com/flutter/devtools/blob/35b74c3198bfaa7ceb386b5fd9acf2cc56067a87/packages/devtools_app/lib/src/screens/memory/memory_controller.dart#L239